moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryIconButtons.js

```diff
@@ -23,18 +23,23 @@ const ProjectSummaryIconButtons = ({
   loading,
 }) => {
   return (
-    <>
+    <div style={{ display: "contents" }}>
```

Comment:
changing this to a div and display: contents makes sure the buttons align correctly in both the project name but also the summary fields.